tests/python/unittest/test_operator.py

```diff
@@ -3365,6 +3368,7 @@ def test_log_softmax():
             check_numeric_gradient(sym, [data], rtol=0.05, atol=1e-3)
 
 
+@attr('nightly')
 def test_pick():
```

Comment:
please leave this test as is. it's an operator that's not in contrib, so we shouldn't leave any chance of breaking it.

Reply:
Roger.